sandbox/rocky/tf/algos/ddpg/ddpg_base_net.py

```diff
@@ -0,0 +1,220 @@
+import tensorflow as tf
+import tensorflow.contrib as tc
+import numpy as np
+
+
+def _fc(x,
```

Comment:
For actor critic, I think we could work on q functions and policies that is implemented directly so they are reusable.